app/lib/govuk_elements_form_builder/form_builder.rb

```diff
@@ -1,106 +1,206 @@
 module GovukElementsFormBuilder
-  class FormBuilder < ActionView::Helpers::FormBuilder
+  # TODO: Refactor this class and remove the rubocop:disable
+  class FormBuilder < ActionView::Helpers::FormBuilder # rubocop:disable Metrics/ClassLength
     # Prevents surrounding of erroneous inputs with <div class="field_with_errors">
     # https://guides.rubyonrails.org/configuring.html#configuring-action-view
     ActionView::Base.field_error_proc = proc { |html_tag| html_tag.html_safe }
 
+    CUSTOM_OPTIONS = %i[label input_prefix field_with_error hide_hint? title inline].freeze
+
     delegate :content_tag, to: :@template
     delegate :errors, to: :@object
 
-    def govuk_text_field(attribute, *args)
-      content_tag :div, class: form_group_classes(attribute) do
-        options = args.extract_options!
+    # Usage:
+    # <%= form.govuk_text_field :name %>
+    #
+    # A hint will be displayed if it is defined in the locale file:
+    # 'helpers.hint.user.name'
+    #
+    # Use the :input_prefix to insert a character inside and at the beginning of the input field.
+    # e.g., <%= form.govuk_text_field :property_value, input_prefix: '$' %>
+    #
+    # Use :field_with_error to have the input be marked as erroneous when an other attribute has an error.
```

Comment:
I've added a :suffix  option in AP-216, so after merging  I'll update this documentation to include that.

Reply:
Cool. Feel free to merge your PR first and I'll modify mine